src/test/fio/ceph-messenger.fio

```diff
@@ -0,0 +1,20 @@
+[global]
+bs=4k
+size=1g
+iodepth=128
+
+ioengine=[path to libfio_ceph_messenger.so]
```

Comment:
I would give a default value that it works like: `ioengine=libfio_ceph_messenger.so`, also you already explain in the documentation that one needs to set LD_LIBRARY_PATH properly

Reply:
Makes sense, updated, thanks.